react/features/mobile/navigation/screenOptions.js

```diff
@@ -273,6 +273,13 @@ export const liveStreamScreenOptions = {
     ...presentationScreenOptions
 };
 
+/**
+ * Screen options for salesforce link modal.
+ */
+export const salesforceScreenOptions = {
```

Comment:
any reason for not directly assigning the object?

Reply:
just copied that part from above and I didn't see the harm to create a new object, fixed it now